### ISSUE TITLE
Deezer extension doesn’t work

### ISSUE THREAD

**Issue by ianhellstrom** [Hammerspoon/hammerspoon#2645]
I have Deezer on my laptop and `hs.deezer.volumeDown()` causes this:

```
...oon.app/Contents/Resources/extensions/hs/deezer/init.lua:253: attempt to perform arithmetic on a nil value
stack traceback:
	...oon.app/Contents/Resources/extensions/hs/deezer/init.lua:253: in function 'hs.deezer.volumeDown'
	(...tail calls...)
	[C]: in function 'xpcall'
	...app/Contents/Resources/extensions/hs/_coresetup/init.lua:514: in function <...app/Contents/Resources/extensions/hs/_coresetup/init.lua:494>
```

Similarly, none of the functions work: `hs.deezer.playpause()` does nothing.

Displaying the current track:

```
2021-01-04 14:24:37: 14:24:37 ERROR:   LuaSkin: Unable to initialize script: {
    NSLocalizedDescription = "A identifier can\U2019t go after this identifier.";
    NSLocalizedFailureReason = "A identifier can\U2019t go after this identifier.";
    OSAScriptErrorAppAddressKey = "<NSAppleEventDescriptor: null()>";
    OSAScriptErrorBriefMessageKey = "A identifier can\U2019t go after this identifier.";
    OSAScriptErrorMessageKey = "A identifier can\U2019t go after this identifier.";
    OSAScriptErrorNumberKey = "-2740";
    OSAScriptErrorOffendingObjectKey = "<NSAppleEventDescriptor: 'utxt'(\"identifier\")>";
    OSAScriptErrorRangeKey = "NSRange: {43, 12}";
}
2021-01-04 14:24:37: 14:24:37 ERROR:   LuaSkin: Unable to initialize script: {
    NSLocalizedDescription = "A identifier can\U2019t go after this identifier.";
    NSLocalizedFailureReason = "A identifier can\U2019t go after this identifier.";
    OSAScriptErrorAppAddressKey = "<NSAppleEventDescriptor: null()>";
    OSAScriptErrorBriefMessageKey = "A identifier can\U2019t go after this identifier.";
    OSAScriptErrorMessageKey = "A identifier can\U2019t go after this identifier.";
    OSAScriptErrorNumberKey = "-2740";
    OSAScriptErrorOffendingObjectKey = "<NSAppleEventDescriptor: 'utxt'(\"identifier\")>";
    OSAScriptErrorRangeKey = "NSRange: {42, 12}";
}
2021-01-04 14:24:37: 14:24:37 ERROR:   LuaSkin: Unable to initialize script: {
    NSLocalizedDescription = "A identifier can\U2019t go after this identifier.";
    NSLocalizedFailureReason = "A identifier can\U2019t go after this identifier.";
    OSAScriptErrorAppAddressKey = "<NSAppleEventDescriptor: null()>";
    OSAScriptErrorBriefMessageKey = "A identifier can\U2019t go after this identifier.";
    OSAScriptErrorMessageKey = "A identifier can\U2019t go after this identifier.";
    OSAScriptErrorNumberKey = "-2740";
    OSAScriptErrorOffendingObjectKey = "<NSAppleEventDescriptor: 'utxt'(\"identifier\")>";
    OSAScriptErrorRangeKey = "NSRange: {42, 12}";
}
```

**Comment by ianhellstrom** [comment]
I believe the issue is related to the fact that Deezer does not have an AppleScript Dictionary: https://support.apple.com/et-ee/guide/script-editor/scpedt1126/2.11/mac/11.0

I tried to run AppleScript directly from the CLI with `osascript -e 'tell application "Deezer" to play'` but that did not work either. So, I tried looking whether the terms had changed only to find out that importing the app into the Script Editor's Library failed with `Unable to add the item because it is not scriptable.`

It might be possible to pass the instructions through `System Events` but that's not the standard way in Hammerspoon AFAICT.

That said, I think the Deezer application should be considered unsupported.
